main.go

```diff
@@ -43,8 +85,16 @@ func main() {
 	)
```

Comment:
This block now feels out of place, it was originally next to the events, but now it's next to SendToSlack. (and now it probably doesn't have such a great name anymore, since there are different types of events.)